playbooks/04-deploy.yaml

```diff
@@ -40,7 +40,3 @@
         name: ceph
         apply:
           become: true
-          delegate_to: >-
-            {{
-              neerali_systems_provisioned | neerali.general.get_admin_nodes
-            }}
```

Comment:
The execution must only happen on the `admin` nodes.

Reply:
understood. We are anyway running this playbook only on `_admin` hosts right. So I thought exclusive delegate is not necessary.

Reply:
Reverted back these changes.